examples/typescript-esm/simple_transfer/transfer.ts

```diff
@@ -0,0 +1,107 @@
+/**
+ * This example shows how to use the Aptos client to create accounts, fund them, and transfer between them.
+ */
+
+import {
+  Account,
+  AccountAddress,
+  Aptos,
+  AptosConfig,
+  Network,
+  SigningScheme,
+  StructTag,
+  TypeTagStruct,
+  U64,
+} from "aptos";
+
+const APTOS_COIN = "0x1::coin::CoinStore<0x1::aptos_coin::AptosCoin>";
```

Comment:
nit
we have this const in `aptos`

Reply:
It's unfortunately not exported.  Will do on another round of exports in a different PR.